### PR TITLE
refactor: Applied most of WebStorm's enhancements.

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/commands/Tools/quote.ts [hunk 1]
```diff
@@ -43,27 +44,29 @@ export default class extends SkyraCommand {
...
+		const guild = this.client.guilds.get(_guild);
+		if (guild !== message.guild!) return null;
 
-			return await (channel as TextChannel).messages.fetch(_message);
-		} catch {
-			return null;
-		}
+		const channel = guild.channels.get(_channel);
+		if (!channel) return null;
+		if (!('messages' in channel)) throw message.language.tget('RESOLVER_INVALID_CHANNEL', 'Channel');
```
Comment (PyroTechniac) [inline src/commands/Tools/quote.ts]:
```suggestion
		if (!(channel instanceof TextChannel)) throw message.language.tget('RESOLVER_INVALID_CHANNEL', 'Channel');
```

**Review by PyroTechniac — CHANGES_REQUESTED** [review]

File: src/commands/Tools/quote.ts (hunk 1 above)
Comment (vladfrangu) [reply src/commands/Tools/quote.ts]: Doesn't account for News channels ^

Comment (PyroTechniac) [reply src/commands/Tools/quote.ts]: Fixed to account for News Channels

Comment (kyranet) [reply src/commands/Tools/quote.ts]: Fixed in e1b69cc

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**